extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -842,11 +842,17 @@ public static Pair<List<ColumnDefinition<?>>, ParquetInstructions> convertSchema
     }
 
     public static final ParquetInstructions LZ4 = ParquetInstructions.builder().setCompressionCodecName("LZ4").build();
+    public static final ParquetInstructions LZ4_RAW =
+            ParquetInstructions.builder().setCompressionCodecName("LZ4_RAW").build();
```

Comment:
should we also test for `LZ4RAW`, since that is apparently the actual name for it?

Reply:
I think the actual name is `LZ4_RAW` only like mentioned [here](https://arrow.apache.org/docs/python/generated/pyarrow.Codec.html) and [here](https://github.com/apache/parquet-format/blob/master/Compression.md). Do you still think we should add tests for `LZ4RAW`?

Reply:
If I'm reading this right, the only reason that LZ4_RAW works at all is the `if` block in `DeephavenCompressorAdapterFactory.getByName` - the underlying hadoop `CompressionCodecFactory.getCodecByName` expects LZ4RAW (well, .toLowerCase())... So I was guessing that something downstream might actually want to pass the string "lz4raw" at some point, so we should validate that it works? 

I've forgotten a lot of the specifics here though, so I'm guessing. I'm not suggesting we have an extra constant here, just something to test that the string Lz4raw still works.

Reply:
Makes sense, thanks for explaining. I have added a test in the python code using string `LZ4RAW`.